src/Repository/Repository.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Repository;
+
+use App\Controllers\Home\HomeController;
+use App\Repository\DBConnexion;
+use PDO;
+
+abstract class Repository
+{
+    protected DBConnexion $dbConnection;
+    protected $table;
+    protected $entity;
+
+    public function __construct(DBConnexion $dbConnection, $table, $entity)
```

Comment:
can you add types, if there are no contract I understand that I can give every things and it mus work ;)